wordle/scratch_3.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import "fmt"
+
+func main() {
+	array1 := []string{"a","b","c","d","e"}
+	array2 := []string{"f","g","h","i","j"}
+	array3 := []string{"k","l","m","n","o"}
+
+
+	var finalArray [][]string
+	finalArray = AppendSlice(finalArray, array1)
+	finalArray = AppendSlice(finalArray, array2)
+	finalArray = AppendSlice(finalArray, array3)
+
+
+	PrintArray(finalArray)
+	PrintArray2(finalArray)
+
+}
+
+func AppendSlice(source [][]string, value []string) [][]string {
+	return append(source, value)
+}
+
+func PrintArray(array [][]string){
+	for i := 0; i < len(array); i++{
+		for j := 0; j <= len(array); j++{
```

Comment:
Length of which array ?

Reply:
This is incorrect implementation. The PrintArray2 function below this is the one that is correct.

Reply:
Why is this included in the PR ?

Reply:
Mistake. I'll overwrite it.

Reply:
I don't like it